app/scripts/modules/core/src/presentation/main.less

```diff
@@ -195,7 +189,7 @@ html {
       margin-left: 0;
     }
     .dl-flex {
-      .dl-wide;
+      .dl-narrow;
```

Comment:
I don't understand what a LESS selector like this does? It doesn't seem to be applying anything. Can you shed some light on it for me?

Reply:
In this case it was controlling 2 different margins in areas of the DOM based on if the filters were expanded or not. `dl-flex` and `dl-narrow` are used in a lot of places so rather than remove this I just controlled streamlined everything to `dl-narrow` in order to have baseline for the rets of the layout changes.

Reply:
my question is about the syntax used.  What does this *do*?
```
.dl-flex {
  .dl-narrow;
}
```

I would expect some rules in side there such as 
```
.dl-flex {
  .dl-narrow {
    width: 12345px
  }
}
```